.gitignore

```diff
@@ -0,0 +1 @@
+/result
```

Comment:
What is `/result`? Can you add a comment?

Reply:
`result` is the default location of where `nix build` writes to.